lib/constants.ts

```diff
@@ -17,6 +17,19 @@ export const NAV_THEME = {
   },
 };
 
+export const SWITCH_THEME = {
```

Comment:
Couldn't we get rid of this if we use the react-native-reusable switch component here? 

https://rnr-docs.vercel.app/components/switch/

Reply:
We are actually using the same 😅 I just added that to get yellow bg instead of the black one (rnr is inspired from shadcn I think)

Reply:
Ah, okay. Can't you update the classes in the Switch component then? (instead of introducing new variables)